### PR TITLE
Libdrs_c

### PR DESCRIPTION
**Pull request by dnadeau4**
Data Retrieval and Storage (DRS) library for climate research

### CODE REVIEW

**Comment by doutriaux1** [comment]
@dnadeau4 now that cdat_info iscin conda-forge you can take out the conflicting files.

**Comment by dnadeau4** [comment]
I'll look at it Friday

**Comment by dnadeau4** [comment]
@ocefpaf What should i do with this conflict?

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,72 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (scopatz) [inline recipes/example/meta.yaml]: Please restore the example recipe.  There is no need to alter any other recipes in the dir than the one in the PR

**Comment by dnadeau4** [comment]
@scopatz I restored the example and the library passed travis CI.  Can we merge into feedstock?

File: recipes/example/meta.yaml [hunk 2]
```diff
@@ -47,8 +47,6 @@ about:
   # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
   # See https://opensource.org/licenses/alphabetical
   license: MIT
-  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
-  license_family: MIT
```
Comment (ocefpaf) [inline recipes/example/meta.yaml]: Why these lines were removed from the example?

File: recipes/libdrs_c/build.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC $CFLAGS"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
```
Comment (ocefpaf) [inline recipes/libdrs_c/build.sh]: Try to preserve to the original flags with `LDFLAGS="-L${PREFIX}/lib $LDFLAGS"`.

File: recipes/libdrs_c/build.sh [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    MAKEFILE=libdrs_Makefile.Linux.gfortran
+    echo "Linux  "${PREFIX}
+else
+    MAKEFILE=libdrs_Makefile.Mac.gfortran
+    echo "Mac  "${PREFIX}
+fi
+cd lib
+sed "s#@cdat_EXTERNALS@#${PREFIX}#g;" ${MAKEFILE}.in > Makefile
+make  
+make  install
```
Comment (ocefpaf) [inline recipes/libdrs_c/build.sh]: `make check`?

File: scripts/run_docker_build.sh [hunk 5]
```diff
@@ -16,6 +16,7 @@ conda-build:
 
 always_yes: true
 show_channel_urls: true
+ssl_verify: false
```
Comment (scopatz) [inline scripts/run_docker_build.sh]: Please don't modify anything outside of the recipe dir in a recipe PR. Thanks!

File: recipes/libdrs_c/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  url: https://github.com/UV-CDAT/libdrs/archive/master.tar.gz
+  sha256: dc057cedbcc0bfcd3aafdc3e04aca9c428f08d6fa1d679778d64be75980462a5
+
+build:
+  number: 0
+  skip: True  # [win] 
+
+requirements:
+  build:
+    - libnetcdf 4.4.1
```
Comment (ocefpaf) [inline recipes/libdrs_c/meta.yaml]: Please check the proper conda-forge pinning: https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies

File: recipes/libdrs_c/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+    - g2clib
+    - gcc
+  run:
+    - libnetcdf 4.4.1
+    - g2clib
+    - gcc
```
Comment (ocefpaf) [inline recipes/libdrs_c/meta.yaml]: You probably don't need `gcc` at run time. I guess that `libgcc` is enough.

File: scripts/run_docker_build.sh (hunk 5 above)
Comment (ocefpaf) [inline scripts/run_docker_build.sh]: Why? We prefer that to be true.

File: recipes/example/meta.yaml (hunk 2 above)
Comment (dnadeau4) [inline recipes/example/meta.yaml]: Not sure what happened here,  I keep merge from master before pushing.  I put the `master` `recipe/example` back.

File: scripts/run_docker_build.sh (hunk 5 above)
Comment (dnadeau4) [inline scripts/run_docker_build.sh]: Will put it back, I have another file now that I run which is not part of the repository.

File: recipes/libdrs_c/meta.yaml (hunk 6 above)
Comment (dnadeau4) [inline recipes/libdrs_c/meta.yaml]: I took it off.

File: scripts/run_docker_build.sh (hunk 5 above)
Comment (dnadeau4) [inline scripts/run_docker_build.sh]: We have a firewall using their own certificate.  This is a pain for every software we use.

File: recipes/libdrs_c/build.sh (hunk 4 above)
Comment (dnadeau4) [inline recipes/libdrs_c/build.sh]: I just looked ans there is not `make check` or no `make test`, this is a very old library for people who use the `drs` file format.

File: scripts/run_docker_build.sh (hunk 5 above)
Comment (ocefpaf) [inline scripts/run_docker_build.sh]: You don't need to run that script. I have the same problem so I use the docker image interactively (which is actually better for debugging): `docker run -it --rm -v $(pwd):/home/ condaforge/linux-anvil /bin/sh`

**Comment by dnadeau4** [comment]
@ocefpaf  I think there is a problem with appveyor!!

**Comment by ocefpaf** [comment]
> @ocefpaf I think there is a problem with appveyor!!

Don't worry. Sometime we get rate limited and AppVeyor gets stuck. But you are not building on Windows here so that is fine. BTW you can commit with `[skip appveyor]` on the message to avoid triggering it.

**Pull request merged**